### PR TITLE
Add language: franch

### PR DESCRIPTION
**Pull request by Gloridust**
Added documentation support for French and added French links to other readme.md files.

### CODE REVIEW

File: docs/cn/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 [![Open In Colab](https://img.shields.io/badge/Colab-F9AB00?style=for-the-badge&logo=googlecolab&color=525252)](https://colab.research.google.com/github/2noise/ChatTTS/blob/main/examples/ipynb/colab.ipynb)
 [![Discord](https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white)](https://discord.gg/Ud5Jxgx5yD)
 
-[**English**](../../README.md) | **简体中文** | [**日本語**](../jp/README.md) | [**Русский**](../ru/README.md) | [**Español**](../es/README.md)
+[**English**](../../README.md) | **简体中文** | [**日本語**](../jp/README.md) | [**Русский**](../ru/README.md) | [**Español**](../es/README.md) | [**Français**](docs/fr/README.md)
```
Comment (fumiama) [inline docs/cn/README.md:15]: Besides EN version `README.md`, the link refer to FR should be `../fr/README.md`. The other files have the same problem.

File: docs/fr/README.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+Un modèle de parole génératif pour le dialogue quotidien.
+
+[![Licence](https://img.shields.io/badge/LICENSE-CC%20BY--NC%204.0-green.svg?style=for-the-badge)](https://github.com/2noise/ChatTTS/blob/main/LICENSE)
+[![PyPI](https://img.shields.io/pypi/v/ChatTTS.svg?style=for-the-badge)](https://pypi.org/project/ChatTTS)
+
+[![Huggingface](https://img.shields.io/badge/🤗%20-Models-yellow.svg?style=for-the-badge)](https://huggingface.co/2Noise/ChatTTS)
+[![Open In Colab](https://img.shields.io/badge/Colab-F9AB00?style=for-the-badge&logo=googlecolab&color=525252)](https://colab.research.google.com/github/2noise/ChatTTS/blob/main/examples/ipynb/colab.ipynb)
+[![Discord](https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white)](https://discord.gg/Ud5Jxgx5yD)
+
+[**English**](../../README.md) | [**简体中文**](docs/cn/README.md) | [**日本語**](docs/jp/README.md) | [**Русский**](docs/ru/README.md) | [**Español**](docs/es/README.md)| **Français**
```
Comment (fumiama) [inline docs/fr/README.md:15]: You can click them to test whether the links are reachable.

**Review by fumiama — CHANGES_REQUESTED** [review]
Thanks!

File: docs/fr/README.md (hunk 2 above)
Comment (Gloridust) [reply docs/fr/README.md:15]: Thanks for pointing out the problem, I will fix it soon

**Comment by Gloridust** [comment]
I committed the fixed version, please check it again.

**Review by fumiama — APPROVED** [review]
No problem now.

**Pull request merged**